Updates/Instances/532_karazhan.sql

```diff
@@ -3396,7 +3324,14 @@ INSERT INTO `spawn_group_entry` (`Id`, `Entry`, `MinCount`, `MaxCount`, `Chance`
 (@SGGUID+23, 181845, 0, 0, 0), (@SGGUID+23, 181846, 0, 0, 0), (@SGGUID+23, 181847, 0, 0, 0), (@SGGUID+23, 181848, 0, 0, 0), (@SGGUID+23, 181768, 0, 0, 0), -- Sealed Tome (Loot)
 (@SGGUID+23, 181833, 0, 0, 2), (@SGGUID+23, 181837, 0, 0, 2), (@SGGUID+23, 181839, 0, 0, 2), (@SGGUID+23, 181841, 0, 0, 2), (@SGGUID+23, 181843, 0, 0, 2), -- Sealed Tome (Trap)
 (@SGGUID+24, 181845, 0, 0, 0), (@SGGUID+24, 181846, 0, 0, 0), (@SGGUID+24, 181847, 0, 0, 0), (@SGGUID+24, 181848, 0, 0, 0), (@SGGUID+24, 181768, 0, 0, 0), -- Sealed Tome (Loot)
-(@SGGUID+24, 181833, 0, 0, 2), (@SGGUID+24, 181837, 0, 0, 2), (@SGGUID+24, 181839, 0, 0, 2), (@SGGUID+24, 181841, 0, 0, 2), (@SGGUID+24, 181843, 0, 0, 2); -- Sealed Tome (Trap)
+(@SGGUID+24, 181833, 0, 0, 2), (@SGGUID+24, 181837, 0, 0, 2), (@SGGUID+24, 181839, 0, 0, 2), (@SGGUID+24, 181841, 0, 0, 2), (@SGGUID+24, 181843, 0, 0, 2), -- Sealed Tome (Trap)
+-- Creatures
+(@SGGUID+50, 15547, 0, 0, 0), (@SGGUID+50, 15548, 0, 0, 0), -- Spectral Charger, Spectral Stallion
+(@SGGUID+52, 15547, 0, 0, 0), (@SGGUID+52, 15548, 0, 0, 0); -- Spectral Charger, Spectral Stallion
+
+INSERT INTO `spawn_group_linked_group` (`Id`, `LinkedId`) VALUES 
+-- Midnight linked trash, attack when midnight gets attacked. 
+(@SGGUID+25, @SGGUID+50), (@SGGUID+25, @SGGUID+51), (@SGGUID+25, @SGGUID+52), (@SGGUID+25, @SGGUID+53),(@SGGUID+25, @SGGUID+54), (@SGGUID+25, @SGGUID+55), (@SGGUID+25, @SGGUID+56), (@SGGUID+25, @SGGUID+57), (@SGGUID+25, @SGGUID+58), (@SGGUID+25, @SGGUID+59), (@SGGUID+25, @SGGUID+60), (@SGGUID+25, @SGGUID+61); 
```

Comment:
This is also not working atm, due to wrong implementation for spawn_group_linked_group
spawn_group_linked aggro only gets trigert when the other group (in this case @sgguid+25) has flag 
CREATURE_GROUP_EVENT_AGGRO
but this group is only the 1 boss mob. Adding CREATURE_GROUP_EVENT_AGGRO to sgguid+25 will crash the server